### DISCUSSION TITLE
Operational Risk

### DISCUSSION THREAD

**Discussion by robmoffat** [risk-first/website#61] [Team Posts]
[Operational Risk](https://github.com/risk-first/website/wiki/Operational-Risk) is a really crucial risk for any software product.  That is, if we expect our software to "live" in the real world.  

The importance of this risk (and the way it has been previously undervalued by developers) is probably why the **DevOps** concept has made such an impact.

I'd like to write something authoritative and engaging on the subject, but I'm not a COO or an Operations Expert by any means, so if anyone feels able to contribute their expertise and ideas, I would be very grateful.  

Read what I've written so far: [Operational Risk](https://github.com/risk-first/website/wiki/Operational-Risk)

thanks.

**Comment by hovancik** [comment]
Great effort, I am currently reading through all the materials and I might have some comments when rereading them.

**Comment by robmoffat** [comment]
great, looking forward to it.  thanks
